src/shared/debug/launchConfiguration.ts

```diff
@@ -51,24 +51,24 @@ export class LaunchConfiguration {
     }
 
     public getDebugConfigurations(): vscode.DebugConfiguration[] {
-        return _(this.configSource.getDebugConfigurations())
-            .map(o => {
-                if (isAwsSamDebugConfiguration(o)) {
-                    ensureRelativePaths(undefined, o)
-                }
-                return o
-            })
-            .value()
+        const configs = this.configSource.getDebugConfigurations()
+        return configs.map(o => {
+            if (isAwsSamDebugConfiguration(o)) {
+                ensureRelativePaths(undefined, o)
+            }
+            return o
+        })
     }
 
     /**
      * Returns all valid Sam Debug Configurations.
      */
     public getSamDebugConfigurations(): AwsSamDebuggerConfiguration[] {
-        return _(this.getDebugConfigurations())
-            .filter(isAwsSamDebugConfiguration)
-            .filter(config => this.samValidator.validate(config)?.isValid)
-            .value()
+        const configs = this.getDebugConfigurations().filter(o =>
+            isAwsSamDebugConfiguration(o)
+        ) as AwsSamDebuggerConfiguration[]
+        const validConfigs = configs.filter(o => this.samValidator.validate(o)?.isValid)
```

Comment:
nit: I think this would read better as one `filter.filter` since we don't care about invalid configs

Reply:
I assume you mean `return configs.filter`.

Reason for assigning it was for debugging; I think in general it helps a lot, and re-adding these lines everywhere is a pain... but I restored `return configs.filter` meanwhile. 

I think this might be a case where overthinking variable names and/or over-affinity for `const` (which really doesn't do much in TS ultimately) is a cost.